app/containers/markdown/components/emoji/Emoji.tsx

```
@@ -1,6 +1,6 @@
 import React, { useContext } from 'react';
-import { Text, useWindowDimensions } from 'react-native';
-import { type Emoji as EmojiProps } from '@rocket.chat/message-parser';
+import { Text, View, useWindowDimensions } from 'react-native';
+import { Emoji as EmojiProps } from '@rocket.chat/message-parser';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)